doc/flame/examples/lib/ray_cast.dart

```diff
@@ -13,7 +13,7 @@ class RayCastExample extends FlameGame with HasCollisionDetection {
   final velocity = 60;
   double get resetPosition => -canvasSize.y;
 
-  Paint paint = BasicPalette.red.withOpacity(0.6).paint();
+  Paint paint = Paint()..color = Colors.red.withOpacity(0.6);
```

Comment:
```suggestion
  final Paint paint = Paint()..color = Colors.red.withOpacity(0.6);
```

Reply:
this is copied as is from main - you can fix on a separate PR!